src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxItem.cs

```diff
@@ -0,0 +1,1100 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Diagnostics;
+using System.Reflection;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Permissions;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Versioning;
+
+namespace System.Drawing.Design
+{
+    /// <summary>
+    /// Provides a base implementation of a toolbox item.
+    /// </summary>
+    [Serializable]
+    public class ToolboxItem : ISerializable
+    {
+        private static TraceSwitch s_toolboxItemPersist = new TraceSwitch("ToolboxPersisting", "ToolboxItem: write data");
```

Comment:
Nit: should this strings be resources?

Reply:
Why would we need to keep resources consistent with .NET Framework? That is not the pattern we follow in .NET Core, we try to make our code base better, we're consistent in behavior, adding this to resources string wouldn't change the behavior.

Reply:
What i meant was, localization team may need to work on this extra string when compared to .NET. But this is not a winforms  specific binary. I will go ahead and add it to resources.